### ISSUE TITLE
Github icon doesn't show up in navbar using wflow_use_github()

### ISSUE THREAD

**Issue by christianholland** [workflowr/workflowr#231]
Hi there,

thanks for this great package. 

TL;DR: Change `fa-github` to `fab fa-github` in `_site.yml`

With `wflow_use_github()` these lines are added automatically to `_site.yml`:

```
  right:
  - icon: fa-github
    href: https://github.com/xxx/yyy
```

which translates in `index.html` to

```
<ul class="nav navbar-nav navbar-right">
        <li>
  <a href="https://github.com/xxx/yyy">
    <span class="fas fa-github"></span>
     
  </a>
</li>
      </ul>
```
However, this leads to the issue that the GitHub logo is not display.

In order to show the logo in `index.html` `fas fa-github` must be replaced by `fab fa-github`. This can be achieved by changing `_site.yml` to 
```
  right:
  - icon: fab-fa-github
    href: https://github.com/xxx/yyy
```

Hence, `wflow_use_github()` must be updated to automatically add the chunk above.

Thanks,
Christian

PS: I am using version 1.6.2

**Comment by jdblischak** [comment]
@christianholland Thank you very much for reporting this so quickly. I confirmed the behavior with rmarkdown 2.6, which was released last week on 2020-12-14. The icon renders fine with rmarkdown 2.5 and earlier. Support for font awesome version 5 was added in [rmarkdown 1.10](https://github.com/rstudio/rmarkdown/blob/master/NEWS.md#rmarkdown-110), but it provided backwards compatibility for font awesome version 4 syntax. I don't see anything in the [NEWS.md](https://github.com/rstudio/rmarkdown/blob/master/NEWS.md#rmarkdown-26) about dropping support for Font Awesome v4, so I suspect this was unintentional. I'll keep investigating to decide how best to address this going forward.
